lang/idris2/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+name                idris2
+github.setup        idris-lang Idris2 0.7.0 v
+categories          lang
+license             BSD
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Purely functional programming language with first class types
+long_description    {*}${description}
+homepage            https://www.idris-lang.org
+checksums           rmd160  0632b400415932d03d750bc6f72b48d8a2d9a6f1 \
+                    sha256  7a8612a1cd9f1f737893247260c6942bf93f193375d4b3df0148f7abf74d6e14 \
+                    size    6987849
+github.tarball_from archive
+
+depends_build-append \
+                    port:chez-scheme \
+                    path:libexec/coreutils/libstdbuf.so:coreutils
+
+depends_lib-append  port:gmp
+
+# Used at runtime too:
+depends_run-append  port:chez-scheme
```

Comment:
being a build- and runtime-dependency is the definition of a "lib" dependency. So just add it once there instead.

Reply:
I do not think it is correct, strictly: lib dependency implies linking. Though I am aware that it is often used that way due to lazy coding :)

Reply:
@reneeotten (No big deal, of course, if you really want lib dep syntax here, I will change that.)

Reply:
@reneeotten Anyway, I changed that to lib, rather save time.